adminguide.rst

```diff
@@ -771,3 +771,44 @@ circular buffer.  For some problems, it may be useful to view this log:
 KVS Eventlogs
 ~~~~~~~~~~~~~
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+flux-accounting Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Creating the flux-accounting database and loading the job priority plugin
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first thing to configure when first setting up the flux-accounting database is to set the PriorityUsageResetPeriod and PriorityDecayHalfLife parameters. Both of these parameters represent a number of weeks by which to hold usage factors up to the time period where jobs no longer play a factor in calculating a usage factor. If these parameters are not passed when creating the DB, PriorityDecayHalfLife is set to 1 week and PriorityUsageResetPeriod is set to 4 weeks, i.e the flux-accounting database will store up to a month's worth of jobs broken up into one week chunks:
+
+.. code-block:: console
+
+ $ flux account create-db --priority-decay-half-life=2 --priority-usage-reset-period=8 path/to/DB
+
+The other component to load is the multi-factor priority plugin, which can be loaded with ``flux jobtap load``:
+
+.. code-block:: console
+
+ $ flux jobtap load mf_priority.so
```

Comment:
Suggestion, also describe how to load the `mf_priority.so` via configuration, which will be how administrators will want to configure the cluster to avoid running manual commands on start/restart.